daemon/endpoint.go

```diff
@@ -57,7 +58,7 @@ func (h *getEndpoint) Handle(params GetEndpointParams) middleware.Responder {
 }
 
 func (d *Daemon) getEndpointList(params GetEndpointParams) []*models.Endpoint {
-	maxGoroutines := 10
+	maxGoroutines := runtime.NumCPU()
```

Comment:
Can you merge these changes into the first commit? I was confused to still see `:= 10` while the commit description talked about max number of cores.

Reply:
Yeah sure, I have updated the pull request.